presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+    post:
+      description: Anonymize image
+      operationId: anonymizeImageRequest
+      parameters:
+      - description: The request body
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/AnonymizeImageRequest'
+      - format: int64
+        in: path
+        name: projectId
+        required: true
+        type: integer
+        x-go-name: ProjectId
+      responses:
+        "200":
+          $ref: '#/responses/anonymizeImageResponse'
+      tags:
+      - anonymize-image
+  /projects/{projectId}/schedule-scanner-cronjob:
+    post:
+      description: Schedule scanner cron-job
+      operationId: scheduleScannerCronjobRequest
+      parameters:
+      - description: The request body
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/ScannerCronJobRequest'
+      - format: int64
+        in: path
+        name: projectId
+        required: true
+        type: integer
+        x-go-name: ProjectId
+      responses:
+        "200":
+          $ref: '#/responses/scheduleScannerCronjobResponse'
+      tags:
+      - schedule-scanner-cronjob
+  /projects/{projectId}/schedule-streams-job:
+    post:
+      description: Schedule streams job
+      operationId: scheduleStreamsJobRequest
+      parameters:
+      - in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/StreamsJobRequest'
+      - format: int64
+        in: path
+        name: projectId
+        required: true
+        type: integer
+        x-go-name: ProjectId
+      responses:
+        "200":
+          $ref: '#/responses/scheduleStreamsJobResponse'
+      tags:
+      - schedule-streams-job
+  /templates/{projectId}/analyze/{templateId}:
+    delete:
+      description: Delete analyze template
+      operationId: deleteAnalyzeTemplateRequest
+      parameters:
+      - in: path
+        name: projectId
+        required: true
+        type: string
+        x-go-name: ProjectId
+      - in: path
+        name: templateId
+        required: true
+        type: string
+        x-go-name: TemplateId
+      responses:
+        "200":
+          $ref: '#/responses/templateGeneralResponse'
+        "400":
+          $ref: '#/responses/templateGeneralError'
+      tags:
+      - template
+      - analyze
+    get:
+      description: Get analyze template
+      operationId: getAnalyzeTemplateRequest
+      parameters:
+      - in: path
+        name: projectId
+        required: true
+        type: string
+        x-go-name: ProjectId
+      - in: path
+        name: templateId
+        required: true
+        type: string
+        x-go-name: TemplateId
+      responses:
+        "200":
+          $ref: '#/responses/templateGeneralResponse'
+        "400":
+          $ref: '#/responses/templateGeneralError'
+      tags:
+      - template
+      - analyze
+    post:
+      description: Create analyze template
+      operationId: postAnalyzeTemplateRequest
```

Comment:
Any chance we call this "createAnalyzeTemplate" instead?
Same with all POST operations that create templates.

Reply:
Sure!
Done